ble/UUID.h

```diff
@@ -29,12 +41,72 @@ class UUID {
         UUID_TYPE_LONG  = 1     // Full 128-bit UUID.
     };
 
+    typedef enum {
+        MSB,
+        LSB
+    } BitOrder_t;
```

Comment:
just a trivial note: this is byte-order, not bit-order. I'll fix this when I merge this PR.